packages/drafts-realm/chess.gts

```diff
@@ -0,0 +1,468 @@
+import {
+  Component as BoxelComponent,
+  CardDef,
+} from 'https://cardstack.com/base/card-api';
+import Component from '@glimmer/component';
+import StringCard from 'https://cardstack.com/base/string';
+import { contains, field } from 'https://cardstack.com/base/card-api';
+
+// @ts-ignore
+import lodash from 'https://cdn.jsdelivr.net/npm/lodash@4.17.21/+esm';
+import {
+  Chessboard,
+  FEN,
+  // @ts-ignore
+} from 'https://cdn.jsdelivr.net/npm/cm-chessboard@8.7.3/+esm';
+
+import Modifier from 'ember-modifier';
+
+type ChessboardModifierSignature = {
+  Args: {
+    Named: {
+      fen?: string;
+    };
+  };
+  Element: HTMLElement;
+};
+
+class ChessboardModifier extends Modifier<ChessboardModifierSignature> {
+  chessboard: any;
```

Comment:
it’s not `Chessboard`?

Reply:
Good suggestion. Which brought me to notice that imports doesn't have types

Reply:
You can manually type it. But yeah, it’s not the best DX…
```ts
let { blah } = await import<BlahType>(“https://blah-es-build”);
```